ui/pages/send/send-content/send-asset-row/send-asset-row.component.js

```diff
@@ -204,11 +212,15 @@ export default class SendAssetRow extends Component {
         onClick={() => this.selectToken(AssetType.native)}
       >
         <div className="send-v2__asset-dropdown__asset-icon">
-          <Identicon
-            diameter={36}
-            image={nativeCurrencyImage}
-            address={nativeCurrency}
-          />
+          {sendAsset?.type === AssetType.NFT && sendAsset?.details?.image ? (
+            <img width={36} src={sendAsset?.details?.image} />
```

Comment:
Since we checked for `sendAsset?.details?.image` in line 215, I don't think we need the optional chaining here.

Reply:
fixed